events/models.py

```diff
@@ -50,6 +50,8 @@ class EventPromotionStatus(Enum):
     in_progress = (20, 'In Progress')
     sent = (30, 'Sent')
     skipped = (40, 'Skipped')
+    event_not_approved = (50, 'Event Not Approved')
+    expired = (60, 'Expired')
```

Comment:
realizing that we made explicit statuses for a couple reasons that event promo will fail, but didn't make statuses for several other reasons it would fail. kind of inconsistent. not sure what is best 🤔

Reply:
i guess its good to have statuses for common fail scenarios that are caused by user error, like event not approved or in the past. and for fail scenarios that are rare or theoretically should never happen i guess we don't need explicit statuses.

another option would be to validate the promotion save and disallow setting promotion to approved for event in past or event not approved 🤔